packages/starlight/__tests__/i18n-non-root-single-locale/translations-fs.test.ts

```diff
@@ -9,7 +9,8 @@ describe('createTranslationSystemFromFs', () => {
 				defaultLocale: { label: 'Français', locale: 'fr', dir: 'ltr' },
 			},
 			// Using non-existent `_src/` to ignore custom files in this test fixture.
-			{ srcDir: new URL('./_src/', import.meta.url) }
+			{ srcDir: new URL('./_src/', import.meta.url) },
+			{}
```

Comment:
Should we make this third parameter optional with a default of `{}` like `createTranslationSystem()` itself to avoid all these pointless extra `{}` in tests? Or do you think it’s too risky we’d forget to pass the parameter in internal usage?